istio/install-istio.go

```diff
@@ -46,6 +49,18 @@ var (
 	bookInfoInjectHTTPAbortToRatingsForJasonFile = path.Join(basePath, "samples/bookinfo/networking/virtual-service-ratings-test-abort.yaml")
 )
 
+func getArch() string {
+	cmd := exec.Command("dpkg", "--print-architecture")
```

Comment:
Wouldnt this fail on a non-debian container? Just a thought. @harsh-not-haarsh @leecalcote

Reply:
@kumarabd How about having a conditional check for debian based os in the function?

Reply:
We do need to first check for presence of `dpkg` and have a backup plan if unavailable.